cfgov/v1/jinja2/v1/includes/organisms/expandable.html

```diff
@@ -39,11 +39,14 @@
      {{ 'itemscope="" itemprop="mainEntity" itemtype="https://schema.org/Question"'
         if value.is_faq else '' }} >
     <button class="o-expandable_header" type="button">
-        <span class="h4 o-expandable_label"
-              {{'itemprop="name"' if value.is_faq else ''}}>
+        <span class="{{ 'h2' if value.larger_heading else 'h4 '}} o-expandable_label"
+              {{'itemprop="name"' if value.is_faq else ''}}
+              {{ 'style="padding:15px 0;"' if value.expanded_padding else ''}}>
+            {% if value.icon %} {{ svg_icon( value.icon ) }} {% endif %} 
             {{ value.label }}
         </span>
-        <span class="o-expandable_cues">
+        <span class="o-expandable_cues"
+        {{ 'style="padding-top:15px;"' if value.expanded_padding else ''}}>
```

Comment:
@anselmbradford here too.

Reply:
Yes this should be added via a modifier on expandables (see other comment).